workflow/rules/common.smk

```diff
@@ -80,10 +80,6 @@ def check_config():
 check_config()
 
 
-def is_activated(config_element):
-    return config_element["activate"] in {"true", "True"}
-
-
 def get_model(wildcards):
```

Comment:
It was always not good that we allowed strings here. The value of activate has to be boolean and we should ensure that via the schema.

Reply:
Already done in response to previous comment.